code/modules/xgm/xgm_gas_mixture.dm

```diff
@@ -519,3 +528,15 @@
 	var/M = get_total_moles()
 	if(M)
 		return get_mass()/M
+
+///Returns a color blended from all materials the gas mixture contains
+/datum/gas_mixture/proc/get_overall_color()
```

Comment:
You mean ``/datum/reagents/proc/get_color()`` right?
I can do that, but /datum/gas_mixture are a lot less well encapsulated than /datum/reagents. A majority of things still change the gas list directly instead of using the procs, and knowing when to clear the cached color is a bit more unreliable.

Reply:
done